pyo3-ffi/src/methodobject.rs

```diff
@@ -71,17 +78,45 @@ extern "C" {
 #[derive(Copy, Clone)]
 pub struct PyMethodDef {
     pub ml_name: *const c_char,
-    pub ml_meth: Option<PyCFunction>,
+    pub ml_meth: PyMethodDefPointer,
     pub ml_flags: c_int,
     pub ml_doc: *const c_char,
 }
 
-impl Default for PyMethodDef {
-    fn default() -> PyMethodDef {
-        unsafe { mem::zeroed() }
-    }
```

Comment:
I don't think this default impl makes sense if PyMethodDefPointer is never null.

Reply:
Agreed; I also actually generally don't like having any implementations of traits or methods for `pyo3-ffi` types if possible (except for `Copy` & `Clone`). Makes it closer to the C that they come from.